.github/workflows/build.yml

```diff
@@ -94,6 +94,36 @@ jobs:
           cp -f $GITHUB_WORKSPACE/app/Source/packages/WorkflowCommons.mpk $GITHUB_WORKSPACE/app/Releases/WorkflowCommons-Latest.mpk
           rm -rf $GITHUB_WORKSPACE/app/Source/packages
 
+      - name: Delete rudiment widgets
+        run: |
+            TEMP_UNZIP_FOLDER="$GITHUB_WORKSPACE/TempUnzip"
+            TARGET_MPK="$GITHUB_WORKSPACE/app/Releases/WorkflowCommons-Latest.mpk"
+            ALLOWED_LIST="$GITHUB_WORKSPACE/app/.github/workflows/AllowedWidgetsList.txt"
+
+            if [! -f "$ALLOWED_LIST" ]; then 
+              echo “Unable to found file AllowedWidgetsList.txt”
+              exit 1; 
+            fi
+
+            if [ -d "$TEMP_UNZIP_FOLDER" ]; then rm -Rf $TEMP_UNZIP_FOLDER; fi
+            mkdir $TEMP_UNZIP_FOLDER
+            unzip -x $TARGET_MPK -d $TEMP_UNZIP_FOLDER
+
+            cd "$TEMP_UNZIP_FOLDER/widgets"
+            for i in *; do
+              if ! grep -q "$i" $ALLOWED_LIST; then
+                echo "Deleting widget: $i" | tee $GITHUB_WORKSPACE/out/deletedWidgets.log
```

Comment:
Currently the deletedWidgets.log file seems to be overwritten for every widget that gets deleted. In the results, the file currently only contains 'Deleting widget: com.mendix.widget.web.DatagridTextFilter.mpk', which is the latest widget that got deleted.